openshift/apigateway.yml

```diff
@@ -0,0 +1,137 @@
+########################################
+## API Gateway ConfigMap
+########################################
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: apigateway
+  namespace: openwhisk
+data:
+  api_host: openwhisk-openwhisk.192.168.64.58.nip.io
```

Comment:
`api_host`, `apigw_url`, and `redis_host` don't need to be hardcoded here. They need to be figured out dynamically in the case of `api-host` and `apigw_url` and perhaps set to just `redis` for redis_host.

Reply:
@bbrowning  - actually I thought to deduce that, but don't know how we can do that - if we can do that then I don't need to  have those `api_host` and `apigw_url` entries.   Thoughts ?